nordvpn/scripts/iptables_rules.sh

```diff
@@ -3,30 +3,30 @@
 INTERFACE_NORD=nordtun
 INTERFACE_LOCAL=eth0
 
-ip -brief addr show $INTERFACE_NORD > /dev/null 2>&1
+ip -brief addr show $INTERFACE_NORD >/dev/null 2>&1
```

Comment:
Nit: Strongly prefer to have a space for `> /dev/null` as in most redirect conventions.